.github/actions/bumpVersion/index.js

```diff
@@ -188,12 +188,17 @@ exports.updateiOSVersion = function updateiOSVersion(version) {
     console.log('Updating iOS', `CFBundleShortVersionString: ${shortVersion}`, `CFBundleVersion: ${cfVersion}`);
 
     // Pass back the cfVersion in the return array so we can set the NEW_IOS_VERSION in ios.yml
-    return Promise.all([
-        cfVersion,
+    const updatePromise = [
         exec(`/usr/libexec/PlistBuddy -c "Set :CFBundleShortVersionString ${shortVersion}" ${PLIST_PATH}`),
         exec(`/usr/libexec/PlistBuddy -c "Set :CFBundleShortVersionString ${shortVersion}" ${PLIST_PATH_TEST}`),
         exec(`/usr/libexec/PlistBuddy -c "Set :CFBundleVersion ${cfVersion}" ${PLIST_PATH}`),
         exec(`/usr/libexec/PlistBuddy -c "Set :CFBundleVersion ${cfVersion}" ${PLIST_PATH_TEST}`),
+    ]
+        .reduce((previousPromise, currentPromise) => previousPromise.then(() => currentPromise()), Promise.resolve());
```

Comment:
Cool! This is sure ugly code :troll: It would be cool to make this into a utility at some point that we can use whenever we need it. Seems like it should work well for you in this case!

Reply:
Another possible cure for this would be to use `execSync()`: https://nodejs.org/api/child_process.html#child_process_child_process_execsync_command_options so that they run synchronously.

Reply:
Oh, and also... I think `exec()` in the array is already starting all the promises all at once. The code you added just makes sure that they are resolved one at a time... So to really get what you want, I think you need to do something more like this:

```
const updatePromises = [
    `/usr/libexec/PlistBuddy -c "Set :CFBundleShortVersionString ${shortVersion}" ${PLIST_PATH}`,
    `/usr/libexec/PlistBuddy -c "Set :CFBundleShortVersionString ${shortVersion}" ${PLIST_PATH_TEST}`,
    `/usr/libexec/PlistBuddy -c "Set :CFBundleVersion ${cfVersion}" ${PLIST_PATH}`,
    `/usr/libexec/PlistBuddy -c "Set :CFBundleVersion ${cfVersion}" ${PLIST_PATH_TEST}`
]
.reduce((previousPromise, command) => previousPromise.then(() => exec(command)), Promise.resolve());;
```

Reply:
Ahhh yeah `execSync` seems like a much better choice for my use case! Thanks!